input/examples/patient-example-sex-and-gender.xml

```diff
@@ -25,7 +29,6 @@
                     <display value="she/her/her/hers/herself"/>
                 </coding>
             </valueCodeableConcept>
-        </extension>
```

Comment:
There is a missing tag in this example I think it is this one - see qa.html

Reply:
This has been addressed, will push shortly